src/common/error/internalError.ts

```diff
@@ -6,40 +6,41 @@ export enum InternalErrorLevel {
     Warning,
 }
 
+export interface IInternalErrorArgument {
```

Comment:
```suggestion
export interface InternalErrorArgument {
```
The reason is that adding `I` for interface names is not a rule in this repo.